staging/src/k8s.io/api/policy/v1beta1/types.go

```diff
@@ -60,6 +60,7 @@ type PodDisruptionBudgetStatus struct {
 	// the list automatically by PodDisruptionBudget controller after some time.
 	// If everything goes smooth this map should be empty for the most of the time.
 	// Large number of entries in the map may indicate problems with pod deletions.
+	// +optional
 	DisruptedPods map[string]metav1.Time `json:"disruptedPods" protobuf:"bytes,2,rep,name=disruptedPods"`
```

Comment:
API Review comment: too late to fix, but this field should have been a list. It is impossible to extend, and the key type is under documented.